image_explorer/public/js/image_explorer.js

```diff
@@ -54,15 +54,35 @@ function ImageExplorerBlock(runtime, element) {
       reveal.css('display', 'block');
       reveal.focus();
       active_feedback = reveal;
+      feedback_open();
       $(this).trigger('feedback:open');
       hotspot_opened_at = new Date().getTime();
       publish_event({
               event_type:'xblock.image-explorer.hotspot.opened',
               item_id: target.parent('.hotspot-container').data('item-id')
       });
     });
+  
+  function pauseVideos(hotspot){
+    // pause any videos playing in this hotspot
+    pauseYoutubeVideos(hotspot);
+    pauseOoyalaVideos(hotspot);
+    pauseBrightcoveVideos(hotspot);
```

Comment:
`hoptspot.find('video').pause()`
This should work for all videos.

Reply:
Yes, but I guess the pause implementation could differ if it's not an HTML5 player? Also for YouTube there's a different method `player.pauseVideo();`

Reply:
Yep, youtube is using HTML5 player underneath. Isn't it?

Reply:
most players will use HTML5 player preferably but they could fallback to other technologies too e.g; Flash, so I think calling player's API method is more safe